Survey/Survey/Sources/Presentation/Views/Buttons/LoginButton.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  LoginButton.swift
+//  Survey
+//
+//  Created by Taher on 3/11/22.
+//
+
+import SwiftUI
+
+struct LoginButton: View {
+    var body: some View {
```

Comment:
Oops, missing line break here

Reply:
add in this [commit](https://github.com/Shayokh144/ic-survey-swiftui-combine/pull/41/commits/7be70826b652894c39041221a9ffeb92e9f7d781)